### PR TITLE
New AngularUI Cheat Sheet

### PR DESCRIPTION
**Pull request by singhbipin2117**
What does your Instant Answer do?

Gives the basic idea about angularui bootstrap
What is the data source for your Instant Answer? (Provide a link if possible)

https://angular-ui.github.io/bootstrap/

Why did you choose this data source?

Its very easy to understand and implement

Are there any other alternative (better) data sources?

Yes

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)

Technology
screenshot
screenshot

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?

No

Are you having any problems? Do you need our help with anything?

NO

### CODE REVIEW

**Comment by daxtheduck** [comment]
@singhbipin2117 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by MariagraziaAlastra** [comment]
Hey @singhbipin2117, thanks for contributing!
Did you know you don't need to fill in the PR template anymore?
Just create an IA Page [here](https://duck.co/ia/new_ia) and then paste the link to it in your first comment :smile: 
You can find more info [in the docs](http://docs.duckduckhack.com/submitting/pull-request.html#submitting-a-pull-request).
Thanks!

**Comment by jophab** [comment]
@singhbipin2117 there is an angularjs cheatsheet already.
I think some aliases  are haviing conflicts
https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/angularjs.json

File: share/goodie/cheat_sheets/json/angularui.json [hunk 1]
```diff
@@ -0,0 +1,163 @@
+{  
+   "id": "angularui_cheat_sheet",
+   "name": "Angularui bootstrap",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/angularui.json]: @singhbipin2117 Lets change this to "AngularUI Bootstrap"

File: share/goodie/cheat_sheets/json/angularui.json [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+   "description": "Explanation of angularui bootstrap",
+   "metadata": {  
+      "sourceName": "Angularui",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/angularui.json]: @singhbipin2117  AngularUI would be better

File: share/goodie/cheat_sheets/json/angularui.json [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+      "sourceUrl": "https://angular-ui.github.io/bootstrap/"
+   },
+   "aliases": [  
+      "js",
+      "java script",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/angularui.json]: @singhbipin2117 lets remove these aliases becaue  they are important ones for js cheatsheet

File: share/goodie/cheat_sheets/json/angularui.json [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+      "Angularjs",
+      "Angularjs Bootstrap",
+      "Angularui",
+      "Angularui Bootstrap"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/angularui.json]: @singhbipin2117 Also avoid use of uppr case lettrs in aliases

**Comment by moollaza** [comment]
@singhbipin2117 thanks again for all the hard work you've put into creating this Angular UI Cheat Sheet. You've done a great job so far! We at DuckDuckGo, and the DuckDuckHack community really appreciate you taking the time to contribute.

It looks like this is just about finished? We still need you to create an Instant Answer page and past the link so this can be deployed to beta.duckduckgo.com for testing. If you're stuck or need any help, I'd love to chat if you're around this week or next. Please message me on slack (@moollaza) or send me an email (moollaza@duckduckgo.com) if you'd be interested in talking. 

Thanks again!

File: share/goodie/cheat_sheets/json/angularui.json [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+   "id": "angularUI_cheat_sheet",
+   "name": "AngularUI Bootstrap",
+   "description": "Explanation of angularUI Bootstrap",
+   "metadata": {  
+      "sourceName": "Angularui",
+      "sourceUrl": "https://angular-ui.github.io/bootstrap/"
+   },
+   "aliases": [
+      "AngularUI",
+      "AngularUI Bootstrap"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/angularui.json]: @singhbipin2117 Lets avoid upper case letters in aliases

File: share/goodie/cheat_sheets/json/angularui.json [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+      "Accordion": [  
+         {  
+            "key": "(ui.bootstrap.accordion)",
+            "val": "The accordion directive builds on top of the collapse directive to provide a list of items"
+         }
+      ],
+      "Alert": [  
+         {  
+            "key": "(ui.bootstrap.alert)",
+            "val": "This directive can be used both to generate alerts from static and dynamic model data "
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/angularui.json]: @singhbipin2117  I think we can shorten the val field a bit.
just specify what it does.
lets avoid "This directive can be used both" from begining.
Check other val fields too for similar correction

**Comment by jdorweiler** [comment]
This pull request is being closed due to inactivity but, if anyone would like to continue it, please @ mention 'duckduckhack-contributors' for help and http://docs.duckduckhack.com/submitting/submitting-overview.html for reference.

**Pull request closed without merging**